src/routes/api/authors.js

```diff
@@ -0,0 +1,30 @@
+const KoaRouter = require('koa-router');
+const JSONAPISerializer = require('jsonapi-serializer').Serializer;
+
+const AuthorSerializer = new JSONAPISerializer('authors', {
+  attributes: ['firstName', 'lastName', 'birthDate'],
+  keyForAttribute: 'camelCase',
+});
+
+const router = new KoaRouter();
+
+router.get('api.authors.show', '/:id', async (ctx) => {
+  const author = await ctx.orm.author.findByPk(ctx.params.id);
+  if (!author) {
+    ctx.throw(404, "The book you are looking for doesen't exists");
```

Comment:
```suggestion
    ctx.throw(404, "The book you are looking for doesn't exist");
```

Reply:
My bad, se me pasó una letra :trollface:

Reply:
De todas formas, esto no es un `show` de _autho_? No debería ser: _The author you are looking for..._

Reply:
También :trollface: